### PR TITLE
[query][api_v2/v_3] Add raw_traces parameter to GetTraceRequest and FindTracesRequest

### PR DESCRIPTION
**Pull request by mahadzaryab1**
## Which problem is this PR solving?
- Towards https://github.com/jaegertracing/jaeger/issues/6417

## Description of the changes
- Added the `raw_traces` parameter to the `GetTraceRequest` and the `TraceQueryParameters` in the `FindTracesRequest`

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [x] I have added unit tests for the new functionality
- [x] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `npm run lint` and `npm run test`

### CODE REVIEW

File: proto/api_v2/query.proto [hunk 1]
```diff
@@ -112,6 +115,7 @@ message TraceQueryParameters {
     (gogoproto.nullable) = false
   ];
   int32 search_depth = 8;
+  bool raw_traces = 9;
```
Comment (yurishkuro) [inline proto/api_v2/query.proto:118]: Same for Find?

Comment (mahadzaryab1) [reply proto/api_v2/query.proto:118]: @yurishkuro this is for `FindTraces`

File: proto/api_v2/query.proto [hunk 2]
```diff
@@ -50,6 +50,9 @@ message GetTraceRequest {
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = false
   ];
+  // Optional. If set to true, the response will not perform any 
+  // adjustments to the trace.
```
Comment (yurishkuro) [inline proto/api_v2/query.proto:54]:
```suggestion
  // enrichments to the trace, such as clock skew adjustment.
  // Instead, the trace will be returned exactly as stored.
```

File: proto/api_v2/query.proto (hunk 1 above)
Comment (yurishkuro) [reply proto/api_v2/query.proto:119]: oh - let's add the same documentation to the field.

Also, why are you limiting this to api_v2 only? Let's make the same change to api_v3

Comment (mahadzaryab1) [reply proto/api_v2/query.proto:122]: @yurishkuro I was going to do v3 in a separate PR but let me know if you prefer it in this one.

Comment (mahadzaryab1) [reply proto/api_v2/query.proto:122]: https://github.com/jaegertracing/jaeger-idl/pull/115

Comment (yurishkuro) [reply proto/api_v2/query.proto:122]: I would do both in one

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**